src/python_testing/TC_LVL_2_3.py

```diff
@@ -189,9 +190,11 @@ async def test_TC_LVL_2_3(self):
         self.step(21)
         remaining_time = sub_handler.attribute_reports[lvl.Attributes.RemainingTime]
         logging.info(f'Reamining time reports: {remaining_time}')
+        asserts.assert_less_equal(remaining_time[0].value, 100, "Unexpected first RemainingTime report")
         asserts.assert_almost_equal(remaining_time[0].value, 100, delta=10, msg="Unexpected first RemainingTime report")
 
         self.step(22)
+        asserts.assert_less_equal(remaining_time[1].value, 150, "Unexpected first RemainingTime report")
```

Comment:
should this be "second" rather than "first" RemainingTIme report?

Reply:
Oh yes, copy paste error 😉 
```suggestion
        asserts.assert_less_equal(remaining_time[1].value, 150, "Unexpected second RemainingTime report")
```